plugins/main/vlan/vlan.go

```diff
@@ -60,10 +62,29 @@ func loadConf(bytes []byte) (*NetConf, string, error) {
 
 func createVlan(conf *NetConf, ifName string, netns ns.NetNS) (*current.Interface, error) {
 	vlan := &current.Interface{}
+	var curNetns ns.NetNS
+	var m netlink.Link
+	var err error
+
+	if conf.MasterInContainer != false {
```

Comment:
checking the "MasterInContainer" flag and applying container network namespace to the find the master and add the vlan link to it.